varcode/variant.py

```diff
@@ -101,8 +109,8 @@ def __init__(self,
         # normalize the variant by trimming any shared prefix or suffix
         # between ref and alt nucleotide sequences and then
         # offset the variant position in a strand-dependent manner
-        trimmed_ref, trimmed_alt, prefix, suffix = trim_shared_flanking_strings(
-            self.original_ref, self.original_alt)
+        (trimmed_ref, trimmed_alt, prefix, suffix) = (
+            trim_shared_flanking_strings(self.original_ref, self.original_alt))
```

Comment:
Gotcha, the actual reason for that change was that the line was 80 chars, which my sublime pep8 plugin was complaining about (it wants <= 79 chars).